app/views/spree/orders/form/_cart_links.html.haml

```diff
@@ -1,5 +1,5 @@
 .row.links
-  %a.continue-shopping.button.secondary{href: current_shop_products_path, "ng-disabled" => "#{@insufficient_stock_lines.any?}", "disable-dynamically" => true}
+  %a.continue-shopping.button.secondary{ @insufficient_stock_lines.any? ? { disabled: "disabled" } : { href: current_shop_products_path } }
     = t :orders_edit_continue
-  %a#checkout-link.button.primary.right{href: main_app.checkout_path, "ng-disabled" => "#{@insufficient_stock_lines.any?}", "disable-dynamically" => true}
+  %a#checkout-link.button.primary.right{ @insufficient_stock_lines.any? ? { disabled: "disabled" } : { href: main_app.checkout_path } }
```

Comment:
Great to use basic html attributes, but I'm not sure this actually disables the element, because `disabled` is not supported on an `<a>` element: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/disabled

But it would be possible to disable with CSS (eg `a[disabled] {pointer-events: none}`). I had a quick look and I couldn't see if there was an existing style rule for that. Can you confirm ❓ 

Or maybe it doesn't matter, if when you click the button, you get a helpful error message. But if that's the case, I'd suggest not using the `disabled` attribute because it doesn't do what it looks like it does.